bsc.verilog/derived_bits/derived_bits.exp

```diff
@@ -51,3 +51,12 @@ compare_verilog mkEnumType2Test.v
 compare_verilog mkEnumType3Test.v
 
 # -----
+
+compile_verilog_pass Foo.bs "" "-opt-undetermined-vals -unspecified-to x"
```

Comment:
Can you give this test a better name than `Foo`?  And can you put a comment here (and maybe in the file) saying what it is testing?  The commit says "an extra test for simplifying expressions involving types with undefined padding bits", but that doesn't get said here or in the source.  (Even having read that comment, I don't quite understand what the example is doing or testing. So I wouldn't object to more comments.)